### PR TITLE
RANCHER-199

### PR DESCRIPTION
**Pull request by Volodymyr-Kartsev**

### CODE REVIEW

File: pipelines/rancher/project.groovy [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!groovy
-@Library('pipelines-shared-library') _
+@Library('pipelines-shared-library@RANCHER-199') _
```
Comment (kdubniak) [inline pipelines/rancher/project.groovy:2]: Need a branch version?

File: pipelines/rancher/psql-dump-restore.groovy [hunk 2]
```diff
@@ -1,24 +1,27 @@
 #!groovy
-@Library('pipelines-shared-library') _
+@Library('pipelines-shared-library@RANCHER-199') _
```
Comment (kdubniak) [inline pipelines/rancher/psql-dump-restore.groovy:2]: the same. Need it change?

File: pipelines/rancher/project.groovy (hunk 1 above)
Comment (Volodymyr-Kartsev) [reply pipelines/rancher/project.groovy]: Fixed

File: pipelines/rancher/psql-dump-restore.groovy (hunk 2 above)
Comment (Volodymyr-Kartsev) [reply pipelines/rancher/psql-dump-restore.groovy]: Fixed

**Review by OHaimanov — APPROVED** [review]

**Review by vkapylou — APPROVED** [review]

**Pull request merged**